README.md

```diff
@@ -122,8 +122,8 @@ There are some special cases where you want to change the default Wabbajack beha
 
 | Flag | Description | Notes |
 |------|-------------|-------|
-| `WABBAJACK_INCLUDE` | All mod files will be inlined into the `.wabbajack` file | |
-| `WABBAJACK_NOMATCH_INCLUDE` | All mod files will be inlined into the `.wabbajack` file even if Wabbajack did not found a match for them | Useful for custom patches you created |
+| `WABBAJACK_INCLUDE` | All mod files will be inlined into the `.wabbajack` file | Can lead to large `.wabbajack` files if used unsparingly |
```

Comment:
```suggestion
| `WABBAJACK_INCLUDE` | All mod files will be inlined into the `.wabbajack` file | **DO NOT USE ON DOWNLOADED MODS** As it would result in you distributing that mod. Can lead to large `.wabbajack` files if used unsparingly |
```